core/controllers/suggestion.py

```diff
@@ -21,13 +21,35 @@
 from constants import constants
 from core.controllers import acl_decorators
 from core.controllers import base
+from core.domain import opportunity_services
 from core.domain import suggestion_services
 from core.platform import models
 import feconf
+import utils
 
 (suggestion_models,) = models.Registry.import_models([models.NAMES.suggestion])
 
 
+def _check_valid_suggestion_type_and_target_type(target_type, suggestion_type):
+    """Checks whether the given target_type and suggestion_type are valid.
+
+    Args:
+        target_type: str. The type of the entity suggestion targets.
```

Comment:
targets --> target. 

Also, what is an "entity suggestion target"? Might be worth explaining.

Reply:
I've made it `The type of suggestion target.`